lib/internal/v8_prof_polyfill.js

```diff
@@ -82,14 +82,15 @@ function readline() {
 }
 
 function versionCheck() {
-  // v8-version looks like "v8-version,$major,$minor,$build,$patch,$candidate"
+  // v8-version looks like
+  // "v8-version,$major,$minor,$build,$patch[,$embedder],$candidate"
   // whereas process.versions.v8 is either "$major.$minor.$build" or
   // "$major.$minor.$build.$patch".
   var firstLine = readline();
   line = firstLine + '\n' + line;
   firstLine = firstLine.split(',');
   const curVer = process.versions.v8.split('.');
-  if (firstLine.length !== 6 && firstLine[0] !== 'v8-version') {
+  if (![6, 7].includes(firstLine.length) || firstLine[0] !== 'v8-version') {
```

Comment:
This is IMO less legible than `(firstLine.length !== 6 && firstLine.length !== 7)`.